<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowEmployee</class>
 <widget class="QDialog" name="WindowEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="Img.qrc">
    <normaloff>:/new/prefix36/img/task.png</normaloff>:/new/prefix36/img/task.png</iconset>
  </property>
  <widget class="QLabel" name="Fon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgb(54, 57, 67);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="upside" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>951</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_PageName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixel Black</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Працівник</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verlayout">
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="employeeName">
       <property name="font">
        <font>
         <family>Fixel Black</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="employeeEmail">
       <property name="font">
        <font>
         <family>Fixel Black</family>
        </font>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: rgb(54, 57, 67);
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnclose">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(://img/white_x_circle_icon.png);
	background-color: transparent;
	background: none;
	border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>63</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {

	image: url(:/new/prefix4/img/icon humanaccount.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_main_task" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>711</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border-radius: 10px;
background-color: rgb(40, 42, 49);
}</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="5,5,5,5,5,5">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:  rgb(30, 101, 172);
border: 2px solid rgb(255, 255, 255);
color:  rgb(255, 255, 255);
border-radius: 14px;
height:  24px;
padding-left: 20px;
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDateEdit" name="dateEdit_6">
       <property name="styleSheet">
        <string notr="true">QDateEdit {
background: rgb(30, 101, 172);
padding: 3px;
color: rgb(255, 255, 255);
font-size: 10;
border: none;
outline: none;
border-radius: 9px;
}

QDateEdit::down-arrow {
image: url(:/new/prefix4/img/calendar_white.png);
width: 30px;
height: 16px;
border: none;
}</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDateEdit" name="dateEdit_5">
       <property name="styleSheet">
        <string notr="true">QDateEdit {
background: rgb(30, 101, 172);
padding: 3px;
color: rgb(255, 255, 255);
font-size: 10;
border: none;
outline: none;
border-radius: 9px;
}

QDateEdit::down-arrow {
image: url(:/new/prefix4/img/calendar_white.png);
width: 30px;
height: 16px;
border: none;
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:  rgb(30, 101, 172);
border: 2px solid rgb(255, 255, 255);
color:  rgb(255, 255, 255);
border-radius: 14px;
height:  24px;
padding-left: 20px;
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="dateEdit_4">
       <property name="styleSheet">
        <string notr="true">QDateEdit {
background: rgb(30, 101, 172);
padding: 3px;
color: rgb(255, 255, 255);
font-size: 10;
border: none;
outline: none;
border-radius: 9px;
}

QDateEdit::down-arrow {
image: url(:/new/prefix4/img/calendar_white.png);
width: 30px;
height: 16px;
border: none;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:  rgb(30, 101, 172);
border: 2px solid rgb(255, 255, 255);
color:  rgb(255, 255, 255);
border-radius: 14px;
height:  24px;
padding-left: 20px;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QCheckBox" name="checkBox_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="styleSheet">
        <string notr="true">QDateEdit {
background: rgb(30, 101, 172);
padding: 3px;
color: rgb(255, 255, 255);
font-size: 10;
border: none;
outline: none;
border-radius: 9px;
}

QDateEdit::down-arrow {
image: url(:/new/prefix4/img/calendar_white.png);
width: 30px;
height: 16px;
border: none;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDateEdit" name="dateEdit_1">
       <property name="styleSheet">
        <string notr="true">QDateEdit {
background: rgb(30, 101, 172);
padding: 3px;
color: rgb(255, 255, 255);
font-size: 10;
border: none;
outline: none;
border-radius: 9px;
}

QDateEdit::down-arrow {
image: url(:/new/prefix4/img/calendar_white.png);
width: 30px;
height: 16px;
border: none;
}</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:  rgb(30, 101, 172);
border: 2px solid rgb(255, 255, 255);
color:  rgb(255, 255, 255);
border-radius: 14px;
height:  24px;
padding-left: 20px;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:  rgb(30, 101, 172);
border: 2px solid rgb(255, 255, 255);
color:  rgb(255, 255, 255);
border-radius: 14px;
height:  24px;
padding-left: 20px;
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:  rgb(30, 101, 172);
border: 2px solid rgb(255, 255, 255);
color:  rgb(255, 255, 255);
border-radius: 14px;
height:  24px;
padding-left: 20px;
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QCheckBox" name="checkBox_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDateEdit" name="dateEdit_3">
       <property name="styleSheet">
        <string notr="true">QDateEdit {
background: rgb(30, 101, 172);
padding: 3px;
color: rgb(255, 255, 255);
font-size: 10;
border: none;
outline: none;
border-radius: 9px;
}

QDateEdit::down-arrow {
image: url(:/new/prefix4/img/calendar_white.png);
width: 30px;
height: 16px;
border: none;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="checkBox_1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Img.qrc"/>
 </resources>
 <connections/>
</ui>
